www/webapp/src/views/Login.vue

```diff
@@ -130,7 +130,7 @@ export default {
           this.$router.replace({ name: 'domains' });
         }
       } catch (ex) {
-        let errors = digestError(ex);
+        let errors = digestError(ex, this);
```

Comment:
there are many more calls to `digestError`, why not adapt all of them?

Reply:
Because the other situations are without a login session (e.g. donation), so the situation that `this` is needed to access the router in order to force an MFA redirect cannot occur.

If we think it's better, I can add it, but I see no benefit.

Reply:
Currently not using the second argument are:

- DonateDirectDebitForm.vue
- ChangeEmail.vue *
- Confirmation.vue 
- TOTPVerifyDialog.vue
- DeleteAccount.vue *
- DynSetup.vue *
- MFA.vue **
- ResetPassword.vue 
- SignUp.vue

*: is used a login token
**: is POSTing to the TOTP verify endpoint (currently) without token

Reply:
done in 0186cf0d9039d133763483e7d091729f00a2cfad, should squash into cd2b1fabaad687d76ad0edf54e5e64476352338c once reviewed.

I added documentation and described the parameter such that when it is not given, the error shall not cause a redirect. So the caller of the error handler can decide whether a redirect would be appropriate in that situation or not.